cli/src/commands/upload.command.ts

```diff
@@ -9,15 +11,23 @@ import { ImmichApi } from 'src/services/api.service';
 import { CrawlService } from '../services/crawl.service';
 import { BaseCommand } from './base-command';
 
+const zipDefined = zip as <T, U>(a: T[], b: U[]) => [T, U][];
+
+enum CheckResponseStatus {
```

Comment:
I think you should be able to just import this type from `@immich/sdk`

Reply:
The server divides the type into action (accept/reject) and reason (duplicate/invalid format), so there's no equivalent for this type in the sdk